swift/ql/lib/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 0.2.0
+
+### Breaking Changes
+
+* The `BraceStmt` AST node's `AstNode getElement(index)` member predicate no longer returns `VarDecl`s after the `PatternBindingDecl` that declares them. Instead, a new `VarDecl getVariable(index)` predicate has been introduced for accessing the variables declared in a `BraceStmt`. This change only affects query writers.
```

Comment:
```suggestion
* The `BraceStmt` AST node's `AstNode getElement(index)` member predicate no longer returns `VarDecl`s after the `PatternBindingDecl` that declares them. Instead, a new `VarDecl getVariable(index)` predicate has been introduced for accessing the variables declared in a `BraceStmt`.
```
This is a library change, so that it only affects query writers is obvious.